### DISCUSSION TITLE
template packages.html improvement

### DISCUSSION THREAD

**Discussion by dalf** [searxng/searxng#3254] [Ideas]
Mock up:

![image](https://github.com/searxng/searxng/assets/1594191/2a382b2c-c8ac-4aee-9f4a-041d1b89f4ee)

Changes:
* there a clear box around each package (like most templates that shows complex results)
* the description is at the bottom like other templates (torrent.html, paper.html, products.html). The description is easy to catch because it is on the bottom right corner, and not sandwitched between the title and the attributes.
* the package name is removed when the title is the package name which avoid duplicate information.

Any interest in that?

**Comment by return42** [comment]
> Any interest in that?

Yes!
